### PR TITLE
Integrate orders and myOrders frontend with backend

### PR DESCRIPTION
**Pull request by bxtsang**

### CODE REVIEW

File: webproject/src/main/webapp/js/room-all-orders.js [hunk 1]
```diff
@@ -7,19 +7,11 @@ window.onload = function() {
...
-            productName: "Fish burger",
-            quantity: 1,
-            perUnitPrice: 15,
-        },
-    ];
+async function getAllOrders() {
+    let response = await fetch(`https://summer20-sps-47.firebaseio.com/orders.json?orderBy=%22roomId%22&equalTo=%22${roomID}%22`)
+    let data = await response.json();
+    let allOrderItems = Object.values(data);
+    console.log(allOrderItems);
```
Comment (kevinluvian) [inline webproject/src/main/webapp/js/room-all-orders.js]: remove console log

File: webproject/src/main/webapp/js/room-my-order.js [hunk 2]
```diff
@@ -6,31 +6,17 @@ window.onload = function() {
...
-        description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s",
-        category: "Food",
-        deliveryFee: "20",
-        minimumOrder: "100",
-        noOfPeopleInRoom: 3,
-    };
+async function getMyOrder() {
+    let response = await fetch(`/myOrder?roomId=${roomID}`);
+    let myOrders = await response.json();
+    console.log(myOrders);
```
Comment (kevinluvian) [inline webproject/src/main/webapp/js/room-my-order.js]: remove console.log

File: webproject/src/main/webapp/js/room-all-orders.js (hunk 1 above)
Comment (arjavibahety) [inline webproject/src/main/webapp/js/room-all-orders.js]: you can remove console.log

File: webproject/src/main/webapp/js/room-my-order.js [hunk 3]
```diff
@@ -6,31 +6,17 @@ window.onload = function() {
...
+    let myOrderItems = Object.values(myOrders);
+
+    let response2 = await fetch(`https://summer20-sps-47.firebaseio.com/rooms/${roomID}.json`)
```
Comment (kevinluvian) [inline webproject/src/main/webapp/js/room-my-order.js]: rename to roomResponse

File: webproject/src/main/webapp/js/room-my-order.js [hunk 4]
```diff
@@ -6,31 +6,17 @@ window.onload = function() {
...
-        shopName: "McDonald's",
-        postalCode: "123456",
-        description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s",
-        category: "Food",
-        deliveryFee: "20",
-        minimumOrder: "100",
-        noOfPeopleInRoom: 3,
-    };
+async function getMyOrder() {
+    let response = await fetch(`/myOrder?roomId=${roomID}`);
```
Comment (kevinluvian) [inline webproject/src/main/webapp/js/room-my-order.js]: rename to orderResponse

File: webproject/src/main/webapp/js/room-my-order.js (hunk 2 above)
Comment (arjavibahety) [inline webproject/src/main/webapp/js/room-my-order.js]: you can remove this

**Review by arjavibahety — CHANGES_REQUESTED** [review]

**Review by kevinluvian — CHANGES_REQUESTED** [review]

**Review by kevinluvian — APPROVED** [review]

**Review by arjavibahety — APPROVED** [review]

**Pull request merged**